src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -174,26 +175,36 @@ export function PanelHeader({
     );
   }
 
+  const renderTitle = () => {
+    const titleComponent = showTitle ? (
+      <span className={title ? 'embPanel__titleText' : 'embPanel__placeholderTitleText'}>
+        {title || placeholderTitle}
+      </span>
+    ) : undefined;
+    return description ? (
+      <EuiToolTip
+        content={description}
+        delay="regular"
+        position="right"
```

Comment:
I would suggest positioning the tooltip to the top otherwise it could be covering vital information from that panel's header.

```suggestion
        position="top"
```

Reply:
Made this change - it works well!